packages/projects-docs/pages/learn/teams/_meta.json

```diff
@@ -1,5 +1,6 @@
 {
   "team-overview": "Working with Teams",
   "permissions": "User Permissions",
-  "preferences": "Preferences"
+  "preferences": "Preferences",
+  "workspace": "Workspace"
```

Comment:
I'm getting a "missing images" error when trying to open this page.

Reply:
maybe it was a bad link? early in the day, I fixed a bunch of links and it seems to work now